client/src/components/entry/SocialSignUp.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import FBIcon from '../../assets/icons/facebook.png';
+import GoogleIcon from '../../assets/icons/google.png';
```

Comment:
should be camelcase to match your other naming?

Reply:
adjustments made